app/views/projects/membership_list.js.erb

```diff
@@ -0,0 +1,10 @@
+$('input[name="authenticity_token"]').val('<%= form_authenticity_token %>');
+$('meta[name="csrf-token"]').attr('content','<%= form_authenticity_token %>');
+$('#flash').html("<%= escape_javascript render partial: "layouts/alert", flash: flash %>");
+$('#hackers').html("<%= escape_javascript render partial: 'hackers' %>");
+$('#membership_buttons').html("<%= escape_javascript render partial: 'membership_buttons' %>");
+$('#file_buttons').html("<%= escape_javascript render partial: 'file_buttons' %>");
```

Comment:
Nitpick: I prefer to use locals and pass the parameter project [like project_info](https://github.com/SUSE/hackweek/pull/305/files#diff-99ad27830ca84e59ef9d59d10ee433c6R10)

Reply:
Why do you prefer them? :-)

Reply:
@hennevogel, for consistency, because you use it in line 10.